### DISCUSSION TITLE
Welcome to leitor-de-texto-em-imagens Discussions!

### DISCUSSION THREAD

**Discussion by afmacaringue** [Makas-M/leitor-de-texto-em-imagens#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fernandogomesfg** [comment]
António, parabéns pela iniciativa!

**Comment by fernandogomesfg** [comment, marked as answer]
Estou interessado em contribuir com o projeto. Para começar, gostaria de saber se há alguma parte específica que você gostaria que eu ajudasse

**Reply by afmacaringue** [reply]
Obrigado por me contactar, vou partilhar os detalhes consigo.
